### PR TITLE
BCN - Vlad & Danielle

### PR DESCRIPTION
**Pull request by danielleleb**

### CODE REVIEW

**Comment by danielleleb** [comment]
@vladcio

File: starter-code/pizza.js [hunk 1]
```diff
@@ -1 +1,72 @@
...
+
+    $pepperonni.toggle();
+    $mushroom.toggle();
+    $greenPepper.toggle();
+
+
+
+}
+
+$(main);
```
Comment (tawebbcn) [inline starter-code/pizza.js:72]: This is a good idea, but you have almost all your code (and therefore your variables) in the global scope. Be careful with that. Better wrap all your code in this function that you will run when de window load (or the document is ready if you use jQuery).

File: starter-code/pizza.js [hunk 2]
```diff
@@ -1 +1,72 @@
...
+
+});
+$mushroomButton.click(function() {
+    $mushroom.toggle();
+    $($activate[1]).toggleClass("active", 'btn btn-mushroom');
+    $($pizzaPrices[1]).toggle()
+
+
+});
+$greenPepperButton.click(function() {
```
Comment (tawebbcn) [inline starter-code/pizza.js:31]: Do you see how you repeat the same code again and again. Maybe you can abstract that into a function.

**Review by tawebbcn — COMMENTED** [review]
Good job! Keep going. :+1:

**Pull request closed without merging**